src/OrchardCore/OrchardCore.Infrastructure.Abstractions/Entities/EntityExtensions.cs

```diff
@@ -77,23 +81,38 @@ public static bool TryGet<T>(this IEntity entity, out T aspect) where T : new()
 
         public static IEntity Put(this IEntity entity, string name, object property)
```

Comment:
`propertyValue`

Reply:
`name` is what we used everywhere else so I like to keep it `name` which is simpler.

Reply:
I didn't mean `name`, we need to change `property` -> `propertyValue` to avoid the confusion

Reply:
We use `property` everywhere else.

Reply:
I know that's why I suggest to rename it while we are on this PR, to make it clear in the future

Reply:
I renamed `property` to `value`.